### PR TITLE
Feature/delete user

### PR DESCRIPTION
**Pull request by mikalojuszrg**

### CODE REVIEW

File: src/pages/Users/UserCard.tsx [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+  const handleDeleteUser = () => {
+    deleteUser(user.id);
+  };
+
   return (
     <Container>
       <Email>{user.email}</Email>
       <p>{getFullName(user.first_name, user.last_name)}</p>
       <p>{hidePassword(user.password)}</p>
+      <AiOutlineDelete onClick={() => handleDeleteUser()} />
```
Comment (rokasandreikenas) [inline src/pages/Users/UserCard.tsx]: `onClick={handleDeleteUser}` jeigu nenaudoji parametru

File: src/api/usersApi.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
-import axios from "axios";
 import { LoginUser, NewUser, User } from "../types/user";
 
-const USERS_API_URL = "https://testapi.io/api/rokasandreikenas/resource/user";
+import axios from "axios";
+
+const USERS_API_URL = "https://testapi.io/api/guodaarr/resource/usersServer";
```
Comment (rokasandreikenas) [inline src/api/usersApi.ts]: parsipullink naujus change, situs reiks ismest

File: src/pages/Users/UserCard.tsx (hunk 1 above)
Comment (mikalojuszrg) [reply src/pages/Users/UserCard.tsx]: updated

File: src/api/usersApi.ts (hunk 2 above)
Comment (mikalojuszrg) [reply src/api/usersApi.ts]: updated

**Pull request merged**